src/WingmanDataTrail/SideBar/SideBar.tsx

```diff
@@ -176,7 +189,13 @@ export class SideBar extends SceneObjectBase<SideBarState> {
               aria-label="Close"
               tooltip="Close"
               tooltipPlacement="top"
-              onClick={() => model.setActiveSection('')}
+              onClick={() => {
+                reportExploreMetrics('metrics_sidebar_toggled', {
+                  action: 'closed',
+                  section: visibleSection.state.key,
+                });
+                model.setActiveSection('');
+              }}
```

Comment:
I suggest undoing this to avoid double-reporting the close action. The action of closing is already caught in the `if (!sectionKey || sectionKey === visibleSection?.state.key) {` block above, where we're already reporting `action: 'closed'`.

```suggestion
              onClick={() => model.setActiveSection('')}
```

Reply:
makes sense - thanks for catching that!